CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Releases
 v1.10.0 (unreleased)
 -------------------
 
+-   Thrift: UberFx-compatible modules are now generated for each service inside
+    a subpackage. Disable this by passing a `-no-fx` flag to the plugin.
```

Comment:
Ideally, we would make this and the mock generator separate plugins but that
makes for a bad UX if we want things to Just Work.

```
THRIFTRW_PLUGINS = go.uber.org/yarpc/encoding/thrift/thriftrw-plugin-yarpc
```

Versus,

```
THRIFTRW_PLUGINS = \
  go.uber.org/yarpc/encoding/thrift/thriftrw-plugin-yarpc \
  go.uber.org/yarpc/encoding/thrift/thriftrw-plugin-gomock \
  go.uber.org/fx/thriftrw-plugin-fx
```

The ship has already sailed on the gomock plugin since it's been out for a
while. For the FX plugin, since our current plan is that you always use YARPC
through Fx, we can probably get away with bundling them together. Especially
because we're adding no new dependencies.

RE: The default value of the flag: It is possible but non-trivial to pass
default arguments to plugins from our internal ThriftRW integration. You would
need to recognize that thriftrw-plugin-yarpc was present in THRIFTRW_PLUGINS
and pass the flag in as `-p "yarpc -fx"`, but only if the version of ThriftRW
being used supports it.